docs/subnets/subnet-cli.md

```diff
@@ -0,0 +1,501 @@
+| PRIMARY P-CHAIN ADDRESS | P-columbus17pysyr6av4n2gf6teqv3kjd5ewdkmncwrhq6qk  |
+*-------------------------*----------------------------------------------------*
+| TOTAL P-CHAIN BALANCE   | 2,499.8990000 $CAM                                 |
+*-------------------------*----------------------------------------------------*
+| URI                     | http://127.0.0.1:9650                              |
+*-------------------------*----------------------------------------------------*
+| NETWORK NAME            | columbus                                           |
+*-------------------------*----------------------------------------------------*
+| CREATED SUBNET ID       | 2dwibTuU3YxvTe7Lc5X4PsmCeC5JiBr1PGJVpCm8T1nec7zUSx |
+*-------------------------*----------------------------------------------------*
+```
+
+### `subnet-cli add validator`
+
+```bash
+subnet-cli add validator \
+--node-ids="[YOUR-NODE-ID]" \
+--stake-amount=[STAKE-AMOUNT-IN-NANO-CAM] \
+--validate-reward-fee-percent=2
+```
+
+To add a validator to the local network:
+
+```bash
+subnet-cli add validator \
+--public-uri=http://localhost:9650NodeID-Nj5C7NdpS3p8hZvu6b5HBREKi8hA1qbDD \  
+--node-ids="NodeID-Nj5C7NdpS3p8hZvu6b5HBREKi8hA1qbDD" \  
+--stake-amount=100000000000 \  
+--validate-reward-fee-percent=3
+```
+
+```text
+2022-07-18T11:00:52.315+0200	info	client/client.go:81	fetching X-Chain id
+2022-07-18T11:00:52.322+0200	info	client/client.go:87	fetched X-Chain id	{"id": "pgk8Re3MCEbQu62orC1ebw7YKzpX1Yk4iS7vTxZReAm9Pjp6m"}
+2022-07-18T11:00:52.322+0200	info	client/client.go:96	fetching CAM asset id	{"uri": "http://localhost:9650"}
+2022-07-18T11:00:52.322+0200	info	client/client.go:105	fetched CAM asset id	{"id": "23MDJv6nhwCGnEer4GsEQWwDURk42ZQBoc444eosZ4BVgH87EW"}
+2022-07-18T11:00:52.322+0200	info	client/client.go:107	fetching network information
+2022-07-18T11:00:52.323+0200	info	client/client.go:116	fetched network information	{"networkId": 1001, "networkName": "columbus"}
+
+Nj5C7NdpS3p8hZvu6b5HBREKi8hA1qbDD is already a validator on 11111111111111111111111111111111LpoYY
+no primary network validators to add
+```
+
+### `subnet-cli add subnet-validator`
+
+```bash
+subnet-cli add subnet-validator \
+--node-ids="[YOUR-NODE-ID]" \
+--subnet-id="[YOUR-SUBNET-ID]"
+```
+
+To add a Subnet validator to the local network:
+
+```text
+> subnet-cli add subnet-validator \
+--private-key-path=.insecure.ewoq.key \
+--public-uri=http://127.0.0.1:12913 \
+--node-ids="NodeID-P7oB2McjBGgW2NXXWVYjV8JEDFoW9xDE5" \
+--subnet-id="24tZhrm8j8GCJRE9PomW8FaeqbgGS4UAQjJnqqn8pq5NwYSYV1"
+```
+
+And its console log:
+
+```text
+2022-05-20T16:10:22.980-0600	info	client/client.go:81	fetching X-Chain id
+2022-05-20T16:10:22.981-0600	info	client/client.go:87	fetched X-Chain id	{"id": "qzfF3A11KzpcHkkqznEyQgupQrCNS6WV6fTUTwZpEKqhj1QE7"}
+2022-05-20T16:10:22.981-0600	info	client/client.go:96	fetching AVAX asset id	{"uri": "http://127.0.0.1:12913"}
```

Comment:
Is this really AVAX on Camino node??

Reply:
No there isn't - this output I missed to replace with the real output from `camino-subnet-cli`